### PR TITLE
TRIVIAL: Fix wrong url in the template

### PR DESCRIPTION
**Pull request by Tulean**

### CODE REVIEW

**Comment by ghost** [comment]
Build succeeded (check pipeline).

**Review by lupko — APPROVED** [review]
LGTM

**Comment by ghost** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-apidocs-fossa-scanning-zuul-el7/4/ : FAILURE in 1m 23s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-apidocs-sonarqube-scan-zuul-gate/8/ : FAILURE in 8m 04s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-apidocs-merge-compliance-check/4/ : SUCCESS in 18s

**Comment by ghost** [comment]
Build succeeded (check pipeline).

**Comment by ghost** [comment]
Build succeeded (check pipeline).

**Comment by ghost** [comment]
Build succeeded (check pipeline).

**Comment by tomaso** [comment]
@martinmacok can you comment on excluding auto-generated html files from sonar scans? It should be fine, but I'd like to involve security team in this.

**Comment by martinmacok** [comment]
@tomaso Yes, it is OK to exclude auto-generated API docs HTML content from SAST analysis.

**Review by lupko — APPROVED** [review]
LGTM

**Comment by ghost** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-apidocs-fossa-scanning-zuul-el7/5/ : FAILURE in 2m 00s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-apidocs-sonarqube-scan-zuul-gate/10/ : FAILURE in 8m 30s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-apidocs-merge-compliance-check/5/ : SUCCESS in 19s

**Comment by ghost** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-apidocs-fossa-scanning-zuul-el7/6/ : FAILURE in 2m 41s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-apidocs-sonarqube-scan-zuul-gate/11/ : FAILURE in 14m 51s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-apidocs-merge-compliance-check/6/ : SUCCESS in 37s

**Comment by ghost** [comment]
Build succeeded (check pipeline).

File: .sonar.settings [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# exclude code generated by a code generator as we are generating 4K+
+# html files in each documentation version.
+sonar.exclusions=**/**/*.html
```
Comment (helobinvn) [inline .sonar.settings]:
```suggestion
sonar.exclusions=**/*.html
```

Comment (helobinvn) [reply .sonar.settings]: and fix the endline error as well

Comment (Tulean) [reply .sonar.settings]: changed

**Comment by ghost** [comment]
Build in pipeline gate aborted.

**Comment by ghost** [comment]
Build succeeded (check pipeline).

**Comment by ghost** [comment]
Build succeeded (check pipeline).

**Comment by yenkins** [comment]
### Sonar scan result
 - No bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ui-apidocs-gate-PR2

**Comment by ghost** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-apidocs-fossa-scanning-zuul-el7/8/ : FAILURE in 1m 50s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-apidocs-sonarqube-scan-zuul-gate/13/ : SUCCESS in 6m 25s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-apidocs-merge-compliance-check/8/ : SUCCESS in 31s

**Pull request merged**